test/unittests/instructions_test.cpp

```diff
@@ -119,6 +119,8 @@ constexpr bool instruction_only_in_evmone(evmc_revision rev, Opcode op) noexcept
     case OP_DATALOADN:
     case OP_DATASIZE:
     case OP_DATACOPY:
+    case OP_TLOAD:
```

Comment:
You could add them to EVMC already

Reply:
I stopped adding these to EVMC (e.g. `MCOPY` is missing). This seems double work with little benefits. Also EVMC instruction information is not directly usable by evmone because not `constexpr`.

What do you think about removing instructions from EVMC? This is a bit of work, but can be done.

Reply:
Probably fine, if nobody uses them.